lib/Backend/GlobOpt.cpp

```diff
@@ -19058,6 +19058,44 @@ GlobOpt::OptDstIsInvariant(IR::RegOpnd *dst)
 }
 
 void
+GlobOpt::OptHoistToLandingPadUpdateValueType(
+    BasicBlock* landingPad,
+    IR::Instr* instr,
+    IR::Opnd* opnd,
+    Value* opndVal)
+{
+    if (instr->m_opcode == Js::OpCode::FromVar)
```

Comment:
I don't understand this...  We copied the FromVar code here.  What is the difference?

---
In reply to: [101593051](https://github.com/Microsoft/ChakraCore/pull/2538#discussion_r101593051) [](ancestors = 101593051,101556159)